### PR TITLE
Wrap `update_by` Exponential Moving Standard Deviation (`emstd`) for python server.

### PR DESCRIPTION
**Pull request by lbooker42**
newplot (3)

newplot (2)

Python code to generate plots:
```
import random
from deephaven import empty_table
from deephaven.updateby import ema_tick, emstd_tick, rolling_avg_tick, rolling_std_tick
from deephaven.plot.figure import Figure

random.seed(77)

random_table = empty_table(1000).update(["idx=ii", "val=(double)random.random() * 1000.0"])

em10 = random_table.update_by([
    ema_tick(10, "ema10=val"), 
    emstd_tick(10, "emstd10=val"),
    rolling_avg_tick("ma10=val", 10), 
    rolling_std_tick("std10=val", 10)
])

figure10 = Figure()
plot10 = figure10.plot_xy(series_name="ema10", t=em10, x="idx", y="ema10") \
        .plot_xy(series_name="ma10", t=em10, x="idx", y="ma10") \
        .plot_xy(series_name="emstd10", t=em10, x="idx", y="emstd10") \
        .plot_xy(series_name="std10", t=em10, x="idx", y="std10") \
        .show()

em50 = random_table.update_by([
    ema_tick(50, "ema50=val"), 
    emstd_tick(50, "emstd50=val"),
    rolling_avg_tick("ma50=val", 50), 
    rolling_std_tick("std50=val", 50)
])

figure50 = Figure()
plot50 = figure50.plot_xy(series_name="ema50", t=em50, x="idx", y="ema50") \
        .plot_xy(series_name="ma50", t=em50, x="idx", y="ma50") \
        .plot_xy(series_name="emstd50", t=em50, x="idx", y="emstd50") \
        .plot_xy(series_name="std50", t=em50, x="idx", y="std50") \
        .show()
```

### CODE REVIEW

File: py/server/deephaven/updateby.py [hunk 1]
```diff
@@ -384,6 +384,78 @@ def emmax_time(ts_col: str, time_scale: Union[int, str], cols: Union[str, List[s
...
+    ticks as the decay unit.
+
+    The formula used is
+        a = e^(-1 / tickDecay)
+        variance = a * (prevVariance + (1 − a) * (x − prevEma)^2)
+        ema = a * prevEma + x
+        std = sqrt(variance)
+
+    Args:
+        time_scale_ticks (int): the decay rate in ticks
```
Comment (jmao-denver) [inline py/server/deephaven/updateby.py:399]: should be `time_scale_ticks (float)`,  the same for the other _ticks functions

**Review by chipkent — CHANGES_REQUESTED** [review]

File: py/server/deephaven/updateby.py [hunk 2]
```diff
@@ -384,6 +384,78 @@ def emmax_time(ts_col: str, time_scale: Union[int, str], cols: Union[str, List[s
...
+            settings as specified in :meth:`~OperationControl.__init__` will be used
+
+    Returns:
+        an UpdateByOperation
+
+    Raises:
+        DHError
+     """
+    try:
+        time_scale = _JDateTimeUtils.expressionToNanos(time_scale) if isinstance(time_scale, str) else time_scale
```
[the remark this answers is not in the document]
Comment (rcaudy) [reply py/server/deephaven/updateby.py:449]: This will certainly merge ahead of the time PR, so we can resolve the issue on that side.

**Review by jmao-denver — APPROVED** [review]
LGTM

**Review by chipkent — APPROVED** [review]

**Comment by deephaven-internal** [comment]
Labels indicate documentation is required. Issues for documentation have been opened:

How-to: https://github.com/deephaven/deephaven.io/issues/2664
Conceptual: https://github.com/deephaven/deephaven.io/issues/2662
Reference: https://github.com/deephaven/deephaven.io/issues/2663
Blog: https://github.com/deephaven/deephaven.io/issues/2661

**Pull request merged**